### ISSUE TITLE
Menu Option for Userlist Toggle

### ISSUE THREAD

**Issue by j-e-n-k-s-y** [chatty/chatty#438]
I use Chatty every day.  Sometimes when flipping through channels, User Lists aren't showing.

SHIFT-F10 does not work on my system, as it's used by other overlays / programs.  As such, I have no ability to toggle the User List on without closing the chat tab and re-joining, which resets to default and shows users.

Two simple solutions exist to this problem.
1) Allow the remapping of the hotkeys for those without ability to use those assumed to be available.
2) Provide menu options to toggle these settings on or off.  

I believe option 2 is easier and meets the needs of my problem.

I have reviewed Github Issue #11 which was closed without any alternatives being provided.

Thank you for your consideration.

**Comment by tduva** [comment]
You can remap all hotkeys in the settings.

**Comment by j-e-n-k-s-y** [comment]
Thanks for pointing that out, @tduva!  I've remapped the key and am successfully toggling the user list, now!

There's a twist(1): What I thought was the userlist being toggled off somehow, was actually it being resized to be 0 pixels wide, and instead just smooshed up against the right side of the window, presumably from being resized and/or dragged from another monitor.

I believe a menu option to toggle the userlist is still a valid request, and would prevent users from having to dig through remapping hotkeys, but I've resolved my own personal issue and will make this as closed.
